bin/idl.js

```diff
@@ -333,6 +329,10 @@ function processArgs(cb) {
         return self.init(cb);
     }
 
+    if (!self.repository) {
```

Comment:
actually the help screen should be invoked:

https://github.com/uber/idl/blob/ee1c242c87d7570b9efb1f68167ae31e4ad15493/bin/idl.js#L320